mp/importer/batcher.py

```diff
@@ -5,32 +5,45 @@
 The user can select the batch size, the maximum number of batches and the starting batch for yielding elements
 Other function will be added to randomly sample elements and to retrieve the last batch  
 """
-def run_in_batches(iterable, end_batch_callback=None, batch_size=2, batch_start=0, max_batches=None):	
+def run_in_batches(iterable, end_batch_callback=None, batch_size=2, batch_start=0, max_batches=None, seed=None, percentage=None):	
     """ Create batches from an iterable
    
     This function is a generator that creates batches of elements from an iterable 
     The callback function is called when a) maximum batches are reached or b) iterable is exhausted 
     All batches (complete and incomplete) are ended by a callback function
+
+    If random options are not None, gets a percentage of the total data
+    Pseudo-samples randomly the elements in the selected percentage (approximate)
+    The main script logs the seed to make it repeatable
     """ 	
+    import random
```

Comment:
in non-test code you should almost always put the imports at the top of the file